### PR TITLE
Fix small heatmap text

### PR DESCRIPTION
**Pull request by mvandenburgh**
Resolves https://github.com/girder/viime/issues/598. Adds another option to the heatmap to force font size to be 12px, with the possible consequence that the user will need to scroll to view the full plot. I called it 'Full', I'm not sure what the best name for it would be, suggestions welcome :).

### CODE REVIEW

File: web/src/components/vis/Heatmap.vue [hunk 1]
```diff
@@ -705,6 +761,19 @@ export default {
...
+  left: 4px;
+  right: 8px;
+  bottom: 8px;
+  display: grid;
+  grid-template-areas: "legend column dl"
+    "row matrix rlabel"
+    "rc clabel ll";
+  justify-content: left;
+  align-content: left;
+}
```
Comment (subdavis) [inline web/src/components/vis/Heatmap.vue:776]: Refactor common styles into a common class, switch to scss.

```scss
.grid {
   ... common styles;
   .grid--large {
      ...large styles that are different;
    }
}
```

Comment (mvandenburgh) [reply web/src/components/vis/Heatmap.vue:767]: @subdavis https://github.com/girder/viime/pull/609/commits/793ef7b50708c4dec35616ac1a5fbe25af82cc8d

**Comment by mvandenburgh** [comment]
old

new

File: web/src/components/vis/Heatmap.vue [hunk 2]
```diff
@@ -664,54 +673,107 @@ export default {
...
-  .rowlabel(ref="rowlabel",
-      :style="{fontSize: fontSize+'px', width: LABEL_WIDTH+'px', height: this.matrixHeight+'px'}",
-      :data-update="reactiveRowLabelUpdate")
-  .legend-wrapper(v-show="columnConfig.dendrogram && rowConfig.dendrogram")
-    .legend(:data-from="legendDomain[0]", :data-to="legendDomain[1]",
-        :style="{background: legendGradient}")
+<template>
+  <div
+    v-resize:throttle="onResize"
+    :class="`grid${layout === 'full' ? ' grid--large' : ''}`"
```
Comment (subdavis) [inline web/src/components/vis/Heatmap.vue:679]: Don't use string interpolation for dynamic classes, there's a better way.

```suggestion
    :class="{
      'grid': true,
      'grid--large': layout === 'full',
    }"
```

**Pull request merged**